Fix:
conftest.c:59:15: error: implicit declaration of function 'realpath' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- configure.orig	2019-12-24 04:13:29.000000000 -0800
+++ configure	2020-11-21 10:30:57.000000000 -0800
@@ -14286,6 +14286,7 @@
 else
   cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
+#include <stdlib.h>

 main(){ (void)realpath("/./tmp", (void *)0); return 0; }

